lib/presentation/widget/upload_request_inside/upload_request_inside_widget.dart

```diff
@@ -515,4 +537,27 @@ class _UploadRequestInsideWidgetState extends State<UploadRequestInsideWidget> {
       .build();
   }
 
+  // Search 
+  Widget _buildSearchViewWidget(BuildContext context) {
```

Comment:
Please move this function to above all action functions. It will make us have consistent code for all widgets.